docs/usage/installation.md

```diff
@@ -2,10 +2,12 @@
 
 ## Requirements
```

Comment:
I think we should distinguish build deployments (NodeJS, OPA, a Rust toolchain) from runtime ones (just a Postgres database), I'll try to clarify that later